libs/static-dynamic-import/users/tsconfig.lib.json

```diff
@@ -8,5 +8,9 @@
     "types": []
   },
   "exclude": ["src/**/*.spec.ts", "jest.config.ts", "src/**/*.test.ts"],
-  "include": ["src/**/*.ts"]
+  "include": [
+    "src/**/*.ts",
+    "../../user/src/lib/user.model.ts",
+    "../../user/src/lib/user.pipe.ts"
```

Comment:
be careful when moving you file, vscode will add the path here

Reply:
I didn't want to have to move the files, but the generate command wouldn't execute inside multiple sub-folders.   I don't have `nx` installed globally.   It is good to know that if you move files, you have to do a cleanup afterwards.

Reply:
I don't think you need nx install globally, npx nx or nx console will work if nx is install inside your project.

Reply:
The generate command changed from `library` to `lib`.  I think I used `library` and it still worked.